mne/simulation/tests/test_metrics.py

```diff
@@ -44,7 +44,8 @@ def test_metrics():
     stc_bad = stc2.copy().crop(0, 0.5)
     assert_raises(ValueError, source_estimate_quantification, stc1, stc_bad)
     stc_bad = stc2.copy()
-    stc_bad.times -= 0.1
+    stc_bad.tmin -= 0.1
+    stc_bad.update_times()
```

Comment:
*we* hack `stc._data` from time to time in the codebase (`stc.crop`, `stc.transform`). I agree that asking users to know about `update_times` is asking too much. Best I can do is refer them to the function in the error message. The behavior in this PR certainly isn't perfect. Question is: is it an improvement on the current situation of no protection whatsoever?